definitions/private-law/json/AuthorisationCaseState/GlobalSearch/AuthorisationCaseState.json

```diff
@@ -0,0 +1,16 @@
+[
```

Comment:
Change this to below format as it is already modified in the FPET-141 branch. Pls change and commit again. 

{
    "CaseTypeID": "PRLAPPS",
    "CaseStateID": "SUBMITTED_PAID",
    "AccessControl": [
      {
        "UserRoles": [
          "caseworker-privatelaw-solicitor",
          "citizen",
          "caseworker-privatelaw-systemupdate",
          "courtnav"
        ],
        "CRUD": "CRUD"
      },
      {
        "UserRoles": [
          "caseworker-privatelaw-courtadmin",
          "caseworker-privatelaw-judge",
          "caseworker-privatelaw-la",
          "caseworker-caa"
        ],
        "CRUD": "CRUD"
      },
      {
        "UserRoles": [
          "caseworker-privatelaw-superuser",
          "caseworker-wa-task-configuration",
          "caseworker-ras-validation"
        ],
        "CRUD": "R"
      }
    ]
  },